sdk/storage/azure-storage-files-datalake/test/ut/datalake_file_system_client_test.cpp

```diff
@@ -220,6 +221,23 @@ namespace Azure { namespace Storage { namespace Test {
       rootPaths.emplace(filename);
     }
 
+    {
+      // startFrom
+      std::set<std::string> results;
+      Files::DataLake::ListPathsOptions options;
```

Comment:
Please add new test cases for new features, don't change exist cases.
Because new feature may not GA everywhere, it may fail during live test.
So we can add `_PLAYBACKONLY_` as suffix of the test name, this test will only run recorded test

And please ensure that each change has the test case coverage, inclue each service, each apis. This can be done in following PR.